llvm/test/CodeGen/SPARC/2011-01-19-DelaySlot.ll

```diff
@@ -60,7 +60,7 @@ entry:
 ;CHECK:      sethi
 ;CHECK:      !NO_APP
 ;CHECK-NEXT: ble
-;CHECK-NEXT: mov
+;CHECK-NEXT: nop
```

Comment:
This test is still correct. The `mov` got hoisted and isn't relevant to this test anymore. There is a `nop` here now which is the point of this test (an instruction is placed in the delay slot).